classes/controller/FrontController.php

```diff
@@ -1454,11 +1454,15 @@ public function setTemplate($default_template)
      * specific controller.
      *
     * @since 1.5.0.13
+    * @param string $defaul_template
     * @return string|bool
     */
-    public function getOverrideTemplate()
+    public function getOverrideTemplate($default_template)
```

Comment:
Hello @cdoublev 
Maybe, you can't change the signature function, so set $default_template to null, ping @maximebiloe